name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/AccountContextMenu.java

```diff
@@ -49,13 +49,6 @@ public void menuAboutToShow(IMenuManager manager, final Account account, final S
 
         manager.add(new Separator());
 
-        new OpenDialogAction(owner, Messages.AccountMenuTransfer) //
```

Comment:
My apologies, I overlooked that this was a cash transfer rather than a security transfer, so you are right, this item should remain.